packages/cli/src/commands/devHandler.js

```diff
@@ -49,8 +51,78 @@ export const handler = async ({
   }
 
   if (side.includes('web')) {
+    let proposedPort = parseInt(getConfig().web.port)
+    const forwardedPortMatches = forward.match(
+      /--port=[0-9][0-9]?[0-9]?[0-9]?[0-9]? ?/
+    )
+    const forwardedPortSet =
+      forwardedPortMatches !== null && forwardedPortMatches.length == 1
+    if (forwardedPortSet) {
+      proposedPort = parseInt(
+        forwardedPortMatches[0]
+          .substring(forwardedPortMatches[0].indexOf('=') + 1)
+          .trim()
+      )
+    }
+
+    const availablePort = await portfinder.getPortPromise({
+      port: proposedPort,
+      stopPort: proposedPort + 64,
+    })
+    if (availablePort != proposedPort) {
```

Comment:
```suggestion
    if (availablePort !== proposedPort) {
```

Reply:
Sorry about the need to pick up these sort of issues. The language I work with day to day doesn't have these sort of two level conditionals so it's just habit of me to read and use `==` instead of `===`. I'll try ensure in the future I take this into account better!